src/gui_common/TweakedColourPicker.cs

```diff
@@ -559,34 +203,6 @@ public override void _Notification(int what)
             base._Notification(what);
         }
 
-        public override void _GuiInput(InputEvent inputEvent)
```

Comment:
It seems you removed the functionality of this child class, but didn't add back anything? To me it looks like this class is entirely non-functional now so this should be entirely deleted to avoid confusion for future programmers who try to reimplement the preset sharing function.